flow360/component/geometry.py

```diff
@@ -199,7 +199,8 @@ def submit(self, description="", progress_callback=None) -> Geometry:
         heartbeat_thread.join()
         ##:: kick off pipeline
         geometry._webapi._complete_upload()
-        log.info("Geometry successfully submitted.")
+        log.info(f"Geometry successfully submitted: {geometry.short_description()}")
+        self._id = info.id
```

Comment:
is the _id of the GeometryDraft used anywhere?

Reply:
yes, it will remove the "remember to submit draft" warning message

Reply:
Ok. Can we add a comment here to indicate this purpose?

Reply:
fixed in: https://github.com/flexcompute/Flow360/pull/526/commits/515cf1ea13850341d9b5ee25da3e4d494f3e7248